webapp/ansible/roles/service-vm/templates/settings.py.j2

```diff
@@ -255,7 +255,7 @@ REST_FRAMEWORK = {
 TEMPORARY_FILE_STORAGE = os.path.abspath("/tmp/lambda_applications")
 
 # Central VM configuration #
-CENTRAL_VM_IP = "{{ central_vm_ip }}"
+CENTRAL_VM_API = "{{ central_vm_ip }}/api"
```

Comment:
I think this should be changed to `CENTRAL_VM_API = "{{ central_vm_host }}/api"`, to match the ansible variable name